deploy/charts/alluxio/templates/helpers/_alluxio-site-properties.tpl

```diff
@@ -46,4 +46,7 @@ alluxio.worker.page.store.type=LOCAL
 alluxio.worker.page.store.dirs=/mnt/alluxio/pagestore
 {{ printf "alluxio.worker.page.store.sizes=%v" .Values.pagestore.quota }}
 
+alluxio.worker.membership.type=ETCD
+{{ printf "alluxio.etcd.endpoints=http://%v-etcd.default.svc.cluster.local:2379" .Release.Name }}
```

Comment:
`default` is the namespace, so it needs to be `.Release.Namespace`. Or just use `%v-etcd` and discard the rest.
Is the port number a configuration in Alluxio? It seems like using host network is not uncommon in k8s, and the port numbers are sometimes dynamic so a static port may not work.

Reply:
no the port is a config in etcd, if not specifically set, it will be 2379 by default, check here:
https://github.com/bitnami/charts/blob/main/bitnami/etcd/templates/statefulset.yaml#L183
and the ports is defined:
https://github.com/bitnami/charts/blob/main/bitnami/etcd/values.yaml#L284
so maybe I should do .Values.containerPorts.client?

Reply:
Right it should be `.Values.etcd.containerPorts.client` because it's a dependency of Alluxio chart.